static/templates/settings/bot-settings.handlebars

```diff
@@ -59,6 +59,13 @@
 
         <div class="form-horizontal" id="api_key_button_box">
             <div class="input-group side-padded-container">
+                <div>
+                    <span>Download single zuliprc of all active bots</span>
```

Comment:
Would it be helpful to say what this is for (e.g. `Download config of all active bots for Flask bot server` or something)?